src/libraries/System.Composition.AttributedModel/src/System.Composition.AttributedModel.csproj

```diff
@@ -1,7 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;net461</TargetFrameworks>
```

Comment:
For this (and all projects) should you also add a corresponding ref project `TargetFramework`?

Reply:
These projects don't have a ref project :)

Reply:
Right you are, I think I mistakenly looked at componentmodel.composition.  I didn't scan every project, but double-check that you matched up the ref frameworks.  That'll make sure you achieve the goal of "not having contributors need to add TargetFrameworks to projects".  If you've already done this feel free to resolve.

Reply:
Yep already done this.